spec/models/like_spec.rb

```diff
@@ -4,9 +4,40 @@
 
 RSpec.describe Like, type: :model do
   let(:like) { FactoryBot.build(:like) }
+  let(:post) { FactoryBot.build(:post) }
+  let(:user) { FactoryBot.build(:user) }
 
   describe 'associations' do
     it { should belong_to(:user) }
     it { should belong_to(:post) }
   end
+
+  # context 'when user likes posts' do
```

Comment:
You ought to remove commented out code.
It’s better to delete commented code. If you need to review a previous version of your code, use git history
This can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).

This is [OPTIONAL] from me since it is a test case but it is generally a requirement for all projects and good practice.